.circleci/config.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           command: |
             . env/bin/activate
             ./bin/sanity.sh 
+            alembic upgrade head
```

Comment:
Won't this run migrations on the `CircleCI` server? I believe we need to add it to our `deploy.sh` script to make it run on the server where it is deployed.

Reply:
You're absolutely right. I missed reading the config thoroughly.